revolv/static/project.scss

```diff
@@ -44,6 +44,317 @@
     }
 }
 
+
+#project_video {
+    margin-top: 30px;
+}
+
+.tabs {
+    margin-left: 4px;
+
+    .tab-item {
+        width: 30%;
+        text-align: center;
+        margin-right: 1%;
+        margin-left: 1%;
+        a {
+            background-color: $revolv-gray-bg;
+            transition: border-color 200ms linear;
+            border-width: 2px;
+            border-style: solid;
+            border-color: $revolv-gray-bg;
+        }
+
+        &.active, &:hover {
+            a {
+                background-color: $revolv-gray-bg;
+                border-color: $revolv-header-color;
+
+            } 
+        }
+    }
+}
+
+#impact-box {
+    border-color: #DCDCDC;
+    border-width: 1px;
+    border-style: solid;
+    height: 250px;
+    width: 100%;
+    margin: 15px 0px 15px 0px;
+
+    h5 {
+        width: 90%;
+        margin-left: 10%;
```

Comment:
Again, these margin lines could be condensed into one `margin: 10px 10%` (`margin: <top and bottom> <left and right>`)